ome-demoserver.yml

```diff
@@ -159,15 +150,15 @@
       become: yes
       pip:
         name:
-        - "omero-figure{{ omero_figure_version_req | default('') }}"
-        - "omero-fpbioimage"
-        - "omero-webtagging-autotag"
-        - "omero-webtagging-tagsearch"
-        - "omero-iviewer"
-        - "omero-parade"
-        - "django-cors-headers"
+        - "omero-figure=={{ omero_figure_release }}"
```

Comment:
Similarly, looks like this says we can no longer pass a comparator.

Reply:
similar to outreach, we want to be explicit with the version we want so user attending a workshop will have the same version installed on demo when they create an account